packages/story-editor/src/utils/isDefaultPage.ts

```diff
@@ -40,18 +37,30 @@ const isDefaultPage = (page) => {
 
   // Check if background color is different
   if (
-    !shallowEqual(
-      page.backgroundColor?.color,
-      defaultPage.backgroundColor?.color
-    )
+    'color' in page.backgroundColor &&
+    'color' in defaultPage.backgroundColor
   ) {
-    return false;
+    if (
```

Comment:
Here, too, lets combine these two `if`s. Alternatively we could return after the first if already. Nested if's like that without any content in between aren't the best. Eg. 
```
if (
!'color' in page.backgroundColor ||
    !'color' in defaultPage.backgroundColor
) { 
return false;
}
// Second if now.
```